Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -455,14 +487,14 @@ protected override void ProcessRecord()
                     else
                     {
                         TablesResource.ListRequest checkTables =
-                            new TablesResource.ListRequest(Service, Project, InputObject.DatasetReference.DatasetId);
+                            new TablesResource.ListRequest(Service, Project, Dataset);
                         var tableResponse = checkTables.Execute();
                         if (tableResponse.TotalItems == 0)
                         {
                             response = request.Execute();
                         }
                         else if (ShouldContinue(
-                            GetConfirmationMessage(InputObject, tableResponse.TotalItems),
+                            GetConfirmationMessage(Dataset, tableResponse.TotalItems),
```

Comment:
Normally if you use ShouldContinue then you should provider a -Force switch. Otherwise, the user has no way of bypassing it (if they are in an environment where they cannot use prompt).